src/runtime/hexagon/hexagon_user_dma_instructions.h

```diff
@@ -24,8 +24,6 @@ namespace tvm {
 namespace runtime {
 namespace hexagon {
 
-#if __HEXAGON_ARCH__ >= 68
```

Comment:
Just a comment---
If someone tries to compile this code for a pre-v68 architecture, the inline assembly will cause an error.  I guess we can require TVM runtime for Hexagon to be compiled for v68 and above, so these guards can remain removed.

Reply:
OK, I will revert the checks for `#if __HEXAGON_ARCH__ >= 68` assuming it's a workable plan to require the TVM runtime to be compiled for v68 and above.

Reply:
Removed.